lib/messagebird/client.rb

```diff
@@ -106,6 +108,18 @@ def otp_verify(recipient, token, params={})
         OTP.new(request(:get, path))
     end
 
+    # Retrieve all messages. Provides the latest message first, unless
+    # ordered_by_received is set to true
+    # Params:
+    # +ordered_by_received+:: boolean - order by received date. Default false.
```

Comment:
I don't think `ordered_by_received` is a good idea. The user of that method that should be responsible for reversing the order if they think its a good idea. Better to keep things simple.